mypy/plugins/attrs.py

```diff
@@ -334,6 +337,10 @@ def _get_converter_name(converter: Optional[Expression]) -> Optional[str]:
                 and converter.node.type.arg_types):
             return converter.node.fullname()
         # Signal that we have an unsupported converter.
+        ctx.api.fail(
+            "Unsupported converter function, only named functions are currently supported",
+            converter
+        )
```

Comment:
For future, we typically don't put parentheses on separate lines (no need to fix this now).